composer.json

```
@@ -1,27 +1,26 @@
 {
   "name": "ibexa/notifications",
-  "license": "proprietary",
+  "license": "GPL-2.0-only",
```

```suggestion
  "license": "(GPL-2.0-only or proprietary)",
```
ref.: https://github.com/ibexa/core/blob/main/composer.json#L5

The context here is that there has to be valid licence both when you install OSS version and paid version that includes that package, hence it's formed like that.